main/javascript-modifications/ses-guide.md

```diff
@@ -0,0 +1,505 @@
+# SES Guide
+
+This document is a guide to understanding *SES (Secure ECMAScript)*. In addition
+to showing what you can and cannot do with a program that implements SES, it gives
+background on the reasons why functionality was added or removed from JavaScript to do so.
+This is intended for initial reading when starting to use or learn about Agoric. For 
+a more limited description of how to use SES without much explanation, see 
+the [SES Reference](./ses-reference.md).
+
+## What is SES
+
+SES (*Secure ECMAScript*) is a JavaScript runtime library for safely running 
+third-party code inside a featherweight compartment. It addresses JavaScript’s 
+lack of security, which is particularly significant in that JavaScript applications
```

Comment:
I'm not really seeing the problem with saying JavaScript has a lack of security; isn't the point of SES to be a secure JavaScript? In this case, I'm referring to "JavaScript" as, well, what you get when you get "all" of JavaScript with SES being a subset of JavaScript with some additions not in JavaScript.  To go back to a previous language I've used, Lisp would be the whole "concept" of Lisp, but Common Lisp would be its own version of Lisp, as would Interlisp. Does anyone else have a problem with the phrasing?

Reply:
I did add in the second paragraph (slightly reworded to tighten). The diagram's worth thinking about, but I'll have to redo it to avoid adding in the several other areas shown in it such as Jessie and JS-Strict that, currently at least, aren't being mentioned in these docs.

Reply:
Perhaps “JavaScript’s lack of internal security”. JavaScript’s claim to fame is that it runs arbitrary code off the internet safely on behalf of a user, so it would be too harsh to say security is entirely absent. It’s in a league of its own compared to C, Python, or…Node.js.